packages/story-editor/src/components/publishModal/content/storyPreview.js

```diff
@@ -192,13 +192,15 @@ const StoryPreview = () => {
     })
   );
 
-  const { allowedImageMimeTypes, hasUploadMediaAction, publisher } = useConfig(
-    ({ allowedMimeTypes, capabilities, metadata }) => ({
-      allowedImageMimeTypes: allowedMimeTypes?.image,
-      hasUploadMediaAction: capabilities?.hasUploadMediaAction,
-      publisher: metadata?.publisher,
-    })
-  );
+  const {
+    allowedImageMimeTypes = [],
+    hasUploadMediaAction,
+    publisher,
+  } = useConfig(({ allowedMimeTypes, capabilities, metadata }) => ({
+    allowedImageMimeTypes: allowedMimeTypes?.image,
+    hasUploadMediaAction: capabilities?.hasUploadMediaAction,
+    publisher: metadata?.publisher,
+  }));
```

Comment:
@littlemilkstudio made a `STABLE_ARRAY` constant that can be used here. Then the children of this component will always see the same reference to that array (if there are no mime types)

```suggestion
  const {
    allowedImageMimeTypes,
    hasUploadMediaAction,
    publisher,
  } = useConfig(({ allowedMimeTypes, capabilities, metadata }) => ({
    allowedImageMimeTypes: allowedMimeTypes?.image || STABLE_ARRAY,
    hasUploadMediaAction: capabilities?.hasUploadMediaAction,
    publisher: metadata?.publisher,
  }));
```